sympy/physics/mechanics/tests/test_actuator.py

```diff
@@ -849,3 +853,144 @@ def test_to_loads(self):
                 diff = (calculated_component - expected_component).subs(substitutions).evalf()
                 # Check if the absolute value of the difference is below a threshold
                 assert Abs(diff) < 1e-9, f"The forces do not match. Difference: {diff}"
+
+class TestCoulombKineticFriction:
+    @pytest.fixture(autouse=True)
+    def _block_on_surface(self):
+        r"""A block sliding on a surface.
+
+        Notes
+        =====
+        This test validates the correctness of the CoulombKineticFriction by simulating
+        a block sliding on a surface with the Coulomb kinetic friction force.
+        The test covers scenarios with both positive and negative velocities.
+
+        Equations of Motion
+        ===================
+        The equations of motion for the block are derived considering
+        the applied force F, the mass m, the gravitational constant g, and the friction forces
+        as described above.
+
+        The general form of the equation of motion is:
+
+            F - F_friction = m * xdd
+
+        where F_friction is given by the friction force equation f_f(v).
+
+        """
+        # Define Dynamics symbols and parameters
+        self.q1, self.u1 = dynamicsymbols("q1 u1", real=True)
+        self.q1_positive, self.u1_positive = dynamicsymbols('q1 u1', real=True, positive=True)
+        self.q1_negative, self.u1_negative = dynamicsymbols('q1 u1', real=True, negative=True)
+        self.repl_positive = {self.q1: self.q1_positive, self.u1: self.u1_positive}
+        self.repl_negative = {self.q1: self.q1_negative, self.u1: self.u1_negative}
+
+        # Mass, gravity constant, friction coefficient, coefficient of Stribeck friction, viscous_coefficient
+        self.m, self.g, self.mu_k, self.mu_s, self.v_s, self.sigma, self.F = symbols('m g mu_k mu_s v_s sigma F', real=True)
+
+        # Define the reference frame
+        self.N = ReferenceFrame('N')
+        self.O = Point('O')
+        self.P = self.O.locatenew('P', self.q1 * self.N.x)
+        self.O.set_vel(self.N, 0)
+        self.P.set_vel(self.N, self.u1 * self.N.x)
+        self.pathway = LinearPathway(self.O, self.P)
+
+        self.particle = Particle('particle', self.P, self.m) # Block
+
+        self.system = System(self.N, self.O)
+        self.system.q_ind = [self.q1]
+        self.system.u_ind = [self.u1]
+        self.system.kdes = [self.q1.diff() - self.u1]
+        self.system.add_bodies(self.particle)
+        self.system.apply_uniform_gravity(-self.g * self.N.y)
+        self.system.add_loads(Force(self.particle, self.F * self.N.x))
+
+    def test_block_on_surface_default(self):
+        friction = CoulombKineticFriction(
+            self.mu_k,
+            self.m * self.g,
+            self.pathway,
+        )
+        self.system.add_actuators(friction)
+
+        # Positive velocity case
+        eoms = self.system.form_eoms()
+        expected_positive = self.F - self.g * self.m * self.mu_k - self.m * self.u1.diff()
```

Comment:
If this is the default where u1 can be positive or negative, where is the `sign(u1)` term?

Reply:
In the setup you have `self.system.q_ind = [self.q1]` where `self.q1, self.u1 = dynamicsymbols("q1 u1", real=True)`, so if you then call `form_eoms()` we should see the `sign(u1)` term present in the equations of motion, right?

Reply:
> If this is the default where u1 can be positive or negative, where is the sign(u1) term?

`sign(u1)` becomes `1` in the positive velocity case. You'll see it when comparing it to the `expected_negative`

Reply:
I'm not seeing where this was made the positive case. You'd have to substitute in the positive assumed symbols before calling `form_eoms()`, right?

Reply:
I agree that the unittest could be rewritten to use `to_loads`. However, the nice thing about testing the positive and negative case separately like this, is that you don't literally write out the same equations as in the source code. In this case one could also have used `Piecewise` or something else (that is of course equivalent) in the source code, which is now not reflected in the test suite.

Reply:
I like the positive/negative test, but it would be much clearer to do:

```
u = dynamicsymbols('u')
pathway = LinearPathway(uses points that have velocities in terms of u)
friction = CoulombKineticFriction(..., pathway)
assert friction.to_loads() == the_correct_result

u = dynamicsymbols('u', positive=True)
pathway = LinearPathway(uses points that have velocities in terms of u)
friction = CoulombKineticFriction(..., pathway)
assert friction.to_loads() == the_correct_result

u = dynamicsymbols('u', positive=False)
pathway = LinearPathway(uses points that have velocities in terms of u)
friction = CoulombKineticFriction(..., pathway)
assert friction.to_loads() == the_correct_result
```

I think this would more directly test what we want to test in this actuator class.